### PR TITLE
feat: define initial event database schemas

### PR DESCRIPTION
**Pull request by matsjla**
Closes: None

## Checklist

- [ ] I have written tests
- [ ] I have provided documentation

## Changelog

- Minimal domain type

## How to test

No lol

### CODE REVIEW

File: packages/ow-db/prisma/schema.prisma [hunk 1]
```diff
@@ -29,3 +29,18 @@ model Company {
   website     String?
   location    String?
 }
+
+model Event {
```
Comment (kharann) [inline packages/ow-db/prisma/schema.prisma:33]: Probably be smart with a 

1. event status, such as Open event, Attendance event, Infinite slots, TBA (we have a lot of TBA events)
2. Eventually, relations to attendance, payment and the users (attendee)

**Comment by kharann** [comment]
Honestly, Would be cool with template events in the future. Like a template for TBA events etc.

File: apps/ow-api/src/modules/event/attendance.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { z } from "zod"
+
+export const attendanceSchema = z.object({
+  id: z.string().uuid(),
+  eventId: z.string().uuid(),
+  start: z.date(),
+  end: z.date(),
+  deregirsterDeadline: z.date(),
```
Comment (kharann) [inline apps/ow-api/src/modules/event/attendance.ts:8]:
```suggestion
  deregisterDeadline: z.date(),
```

**Review by kharann — CHANGES_REQUESTED** [review]

**Comment by kharann** [comment]
Also, thoughts on `eventId` vs `eventID`

**Comment by matsjla** [comment]
I like eventId because eventID just looks weird :c

**Comment by kharann** [comment]
f a i r

**Review by kharann — APPROVED** [review]

**Pull request merged**